CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Logic Database
 
+##### Tallon Overworld
+
+- Added: Reverse Frigate now requires Knowledge (Beginner) in every room where the door usually requires activating Thermal Conduits from the other side.
 - Changed: The Varia-only heat reduction is now done via a miscellaneous resource rather than being patched at runtime.
```

Comment:
```suggestion
- Changed: The Varia-only heat reduction is now done via a miscellaneous resource rather than being patched at runtime.

##### Tallon Overworld

- Added: Reverse Frigate now requires Knowledge (Beginner) in every room where the door usually requires activating Thermal Conduits from the other side.
```

Reply:
oh whoops good catch